spec/index.js

```diff
@@ -11,7 +11,7 @@ fs.readdirSync(testDir).filter((file) => file.includes(specModule + '.spec.js'))
       mocha.addFile(path.join(testDir, file));
 
 });
-mocha.timeout(20000);
+mocha.timeout(22000);
```

Comment:
If possible, could you please add some information to this pull request by explaining why you had to increase this timeout? If you can easily find the failing test case for the previous timeout, you can mention it too.